app/controllers/idv/gpo_verify_controller.rb

```diff
@@ -85,20 +85,20 @@ def prepare_for_personal_key
       idv_session.address_confirmed!
     end
 
-    def throttle
-      @throttle ||= Throttle.new(
+    def rate_limiter
+      @rate_limiter ||= RateLimiter.new(
         user: current_user,
-        throttle_type: :verify_gpo_key,
+        rate_limit_type: :verify_gpo_key,
       )
     end
 
-    def render_throttled
+    def render_rate_limited
       irs_attempts_api_tracker.idv_gpo_verification_rate_limited
       analytics.throttler_rate_limit_triggered(
         throttle_type: :verify_gpo_key,
       )
 
-      @expires_at = throttle.expires_at
+      @expires_at = rate_limiter.expires_at
       render :throttled
```

Comment:
to be changed in follow up ticket/PR?

Reply:
Yes, I was avoiding changing anything route-related. Could this be changed by changing just the name of the template file? I still think it should be in a followup PR.